deployment/helm/node-feature-discovery/Chart.yaml

```diff
@@ -13,3 +13,7 @@ keywords:
   - node-labels
 type: application
 version: 0.2.1
+dependencies:
+  - name: spire
+    version: 0.24.1
+    repository: https://spiffe.github.io/helm-charts-hardened/
```

Comment:
We should have a conditional for this. Only require if spiffe is enabled. Right(?)